s3recovery/s3recovery/s3recoverybase.py

```diff
@@ -297,10 +317,6 @@ def dry_run(self, index_name, index_id, index_id_replica, union_result, recover_
 
             self.perform_validation(key, metadata_value, replica_value, union_result)
 
-            # Perform cleanup for existing indices during recovery
-            if (recover_flag):
```

Comment:
recover_flag is redundant - let us remove

Reply:
Done